### PR TITLE
Feature/non cash contributions

### PR DESCRIPTION
**Pull request by PushBackNeil**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
 
 
+  namespace :project do
```
Comment (stuartmccoll) [inline config/routes.rb]: Not sure what's happened here, but we've ended up with two `:project` namespaces.

Looks like this already exists in the other `:project` namespace, albeit the method being called is one character different.

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -23,10 +23,5 @@
...
 
-// ############################################################################
-// NLHF SCSS Imports (namespace 'nlhf-')
-// ############################################################################
-@import "helpers/all";
-@import "tools/all";
-@import "components/all";
-@import "utilities/all";
+// other NLHF imports below here
+@import "nlhf";
```
Comment (stuartmccoll) [inline app/assets/stylesheets/application.scss]: Has something gone wrong with merging `master` in here? Looking at the latest copy of `master`, this seems to be taking @paulmsmith's changes back out.

File: app/assets/stylesheets/_nlhf.scss [hunk 3]
```diff
@@ -0,0 +1,290 @@
+
```
Comment (stuartmccoll) [inline app/assets/stylesheets/_nlhf.scss]: Looks like this is reverting @paulmsmith's changes from `master` too.

File: config/routes.rb (hunk 1 above)
Comment (PushBackNeil) [reply config/routes.rb]: @stuartmccoll good spot, rails g strikes again!

File: app/assets/stylesheets/_nlhf.scss (hunk 3 above)
Comment (PushBackNeil) [reply app/assets/stylesheets/_nlhf.scss]: Looks like the merge from master went wrong, I will fix it.

Comment (PushBackNeil) [reply app/assets/stylesheets/_nlhf.scss]: @paulmsmith its strange the files Paul added were there.

**Pull request merged**